python/ray/worker.py

```diff
@@ -1088,7 +1101,9 @@ def get_address_info_from_redis_helper(redis_address, node_ip_address):
     client_info = {"node_ip_address": node_ip_address,
                    "redis_address": redis_address,
                    "object_store_addresses": object_store_addresses,
-                   "local_scheduler_socket_names": scheduler_names}
+                   "local_scheduler_socket_names": scheduler_names,
+                   # Web UI should be running.
+                   'webui_url': webui_url_helper(redis_client)}
```

Comment:
Please be consistent with the surrounding code, so if all of the keys are strings using double quotes then when you add a new key it should be formatted the same way.